cmd/thanos/tools_bucket.go

```diff
@@ -997,8 +997,8 @@ func (t Table) Less(i, j int) bool {
 
 func compare(s1, s2 string) bool {
 	// Values can be either Time, Duration, comma-delimited integers or strings.
-	s1Time, s1Err := time.Parse("02-01-2006 15:04:05", s1)
-	s2Time, s2Err := time.Parse("02-01-2006 15:04:05", s2)
+	s1Time, s1Err := time.Parse(time.RFC1123, s1)
```

Comment:
Does this need to be `RFC3339`?

Reply:
Good catch 😆  I'm not sure how I mixed up my copy/paste there but fixed now 🙇